src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const basePrice = 40;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (days >= LONG_TERM) {
+    return days * basePrice - LONG_TERM_DISCOUNT;
```

Comment:
```suggestion
    return days * basePrice - LONG_TERM_DISCOUNT;
```
return days * basePrice can be a variable